src/main/cpp/blaze_util_windows.cc

```diff
@@ -536,20 +540,20 @@ static void CreateCommandLine(CmdLine* result, const string& exe,
     }
 
     if (has_space) {
-      cmdline << '\"';
+      cmdline << L'\"';
     }
   }
 
-  string cmdline_str = cmdline.str();
+  wstring cmdline_str = cmdline.str();
   if (cmdline_str.size() >= MAX_CMDLINE_LENGTH) {
     BAZEL_DIE(blaze_exit_code::INTERNAL_ERROR)
         << "Command line too long (" << cmdline_str.size() << " > "
-        << MAX_CMDLINE_LENGTH << "): " << cmdline_str;
```

Comment:
Can you convert `cmdline_str` to a `string` and not removing it from here? It will give better error message.
`blaze_util::WstringToCstring()` ?

Reply:
Done.